### PR TITLE
add coin/fa minting benchmarks

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

File: testsuite/single_node_performance.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```
Comment (msmouse) [inline testsuite/single_node_performance.py:1]: ? why

File: testsuite/single_node_performance.py [hunk 2]
```diff
@@ -31,6 +31,7 @@ class Flow(Flag):
     AGG_V2 = auto()
     # Test resource groups
     RESOURCE_GROUPS = auto()
+    TEST = auto()
```
Comment (msmouse) [inline testsuite/single_node_performance.py:34]: not used?

File: testsuite/single_node_performance.py [hunk 3]
```diff
@@ -136,6 +137,11 @@ class RunGroupConfig:
     RunGroupConfig(expected_tps=1300, key=RunGroupKey("token-v1nft-mint-and-transfer-parallel"), included_in=Flow(0)),
     RunGroupConfig(expected_tps=5300, key=RunGroupKey("token-v1nft-mint-and-transfer-parallel", module_working_set_size=20), included_in=Flow(0)),
 
+    RunGroupConfig(expected_tps=14000, key=RunGroupKey("coin-init-and-mint", module_working_set_size=1), included_in=LAND_BLOCKING_AND_C),
```
Comment (msmouse) [inline testsuite/single_node_performance.py:140]: LAND_BLOCKING_AND_C not defined?

File: testsuite/single_node_performance.py [hunk 4]
```diff
@@ -419,17 +425,17 @@ def print_table(
...
+    # try:
+    #     execute_command(f"RUST_BACKTRACE=1 {BUILD_FOLDER}/aptos-move-e2e-benchmark")
+    #     move_e2e_benchmark_failed = False
+    # except:
+    #     # for land-blocking (i.e. on PR), fail immediately, for speedy response.
+    #     # Otherwise run all tests, and fail in the end.
+    #     if SELECTED_FLOW == Flow.LAND_BLOCKING:
+    #         print("Move E2E benchmark failed, exiting")
+    #         exit(1)
+    #     move_e2e_benchmark_failed = True
```
Comment (msmouse) [inline testsuite/single_node_performance.py:438]: ?

File: testsuite/module-publish/src/packages/framework_usecases/sources/coin_example.move [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        aptos_framework::managed_coin::initialize<ExampleCoin>(
+            sender,
+            b"Example Coin",
+            b"Example",
+            8,
+            false,
+        );
+    }
+
+    public entry fun mint_p(user: &signer, admin: &signer, amount: u64) {
```
Comment (msmouse) [inline testsuite/module-publish/src/packages/framework_usecases/sources/coin_example.move:16]: sorry but what does `_p` stand for and how does an entry function take two signers?

Comment (igor-aptos) [reply testsuite/module-publish/src/packages/framework_usecases/sources/coin_example.move:16]: I saw someone else use _p before, so I continued using it. it is for "payer" before we had a proper fee payer.

but basically here it is for indirection - we sign a transaction with two signers - user is used for seq_num and gas, and admin is a second signature, for authorization purposes

File: testsuite/single_node_performance.py (hunk 3 above)
Comment (igor-aptos) [reply testsuite/single_node_performance.py:139]: it's defined as:

LAND_BLOCKING_AND_C = Flow.LAND_BLOCKING | Flow.CONTINUOUS

Comment (msmouse) [reply testsuite/single_node_performance.py:139]: hmm.. where? :trollface:

**Review by msmouse — APPROVED** [review]

File: testsuite/single_node_performance.py (hunk 3 above)
Comment (igor-aptos) [reply testsuite/single_node_performance.py:139]: right below the enum - https://github.com/aptos-labs/aptos-core/blob/main/testsuite/single_node_performance.py#L37

Comment (msmouse) [reply testsuite/single_node_performance.py:139]: My IDE sucks (no, I suck, I searched in '*.rs')

**Review by sitalkedia — APPROVED** [review]

**Pull request merged**